index.html

```diff
@@ -126,7 +126,8 @@ <h2>
         Examples
       </h2>
 
-      <pre class="example" title="Example did:web DID document using JSON Web Keys">
+      <pre class="example"
```

Comment:
nit: Theres quite a bit of linting changes going on, do we have a linter in the repo or similar?

Reply:
i think that is on my side